template/Content/FeatherHttpContent.csproj

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk.Web">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="FeatherHttp" Version="TheNugetPackageVersion" />
```

Comment:
Maybe a variable? Even if its not specified to make the project file valid syntax?

Reply:
I've changed this to use the $(PackageVersion) supplied by the nerdbrain - even though msbuild is not replacing this on a `dotnet pack` it should still work if you do a straight build of that project